go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/kubernetes-csi/csi-test/v3
 go 1.12
 
 require (
-	github.com/container-storage-interface/spec v1.2.0
+	github.com/container-storage-interface/spec v1.2.0-rc1.0.20200530102742-4c5274ab4e5e
```

Comment:
You can use this one:
https://github.com/container-storage-interface/spec/releases/tag/v1.3.0-rc1

Reply:
Got it.